client/src/pages/offering.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+
+import Button from "../components/Button";
+import CardOffering from "../components/CardOffering";
+import ContainerFlat from "../components/ContainerFlat";
+
+import search from "../assets/images/search.svg";
+
+export default function Offering() {
+  return (
+    <>
+      <ContainerFlat>
+        <Button_centered> add new gadg</Button_centered>
+
+        <H3>my offered gadges</H3>
+        <Search src={search} />
+
+        <Flexbox>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+          <CardOffering>map...</CardOffering>
+        </Flexbox>
+      </ContainerFlat>
+    </>
+  );
+}
+
+const H3 = styled.h3`
+  text-align: center;
+  margin: 0 auto;
+`;
+
+const Button_centered = styled(Button)`
+  width: 30%;
+  margin: 1rem auto;
```

Comment:
When you let the ContainerFlat adjust its children, maybe with `display: grid` you dont need to use so much margins